msg/estimator_aid_source_1d.msg

```diff
@@ -0,0 +1,21 @@
+uint64 timestamp             # time since system start (microseconds)
+uint64 timestamp_sample      # the timestamp of the raw data (microseconds)
+
+uint8 estimator_instance
+
+uint32 device_id
+
+uint64 time_last_fuse
```

Comment:
Should we also include `time_last_data`? That's also something we're using for most of the aiding sources in order to disable the fusion in case of a sensor timout.

Reply:
I actually had that initially, but at the moment I can't remember the reason it didn't quite fit.

I viewed this is as a small step towards gathering up everything to do with a particular aid source into one structure (estimator_aid_source, RingBuffer, noise parameters, delay, time_last_data, etc) that we'd ultimately allocate per sensor instance (based on configuration). Should we keep iterating in that direction?